test-clients/src/main/java/com/hedera/services/bdd/suites/utils/validation/ValidationScenarios.java

```diff
@@ -253,7 +241,7 @@ private static HapiApiSpec ensureBytecode() {
 							withOpContext((spec, opLog) -> {
 								if (feeSnapshots.getOpsConfig().getBytecode() == null) {
 									var bytecodeCreate = fileCreate("unusedName")
-											.fromResource("Multipurpose.bin");
```

Comment:
When `ValidationScenarios#main` is run as a standalone JAR, it can't read files from _src/main/resource_, but needs to load them via `ClassLoader.getResourceAsStream(String)`. 

This doesn't really matter since `ValidationScenarios` is deprecated and will not be ever be enhanced or a new version given to DevOps, but just an aside. :)